docs/docs/cheat-sheet.md

```diff
@@ -0,0 +1,283 @@
+---
+title: Cheat Sheet
+---
+
+The Gatsby team has created a resource that you might find useful when building a Gatsby site: a cheat sheet with all the top commands and development tips! Feel free to download it and reference digitally or print yourself a copy. For related online information, visit the [Quick Start](/docs/quick-start/) page.
```

Comment:
perhaps we should say eg: `print yourself a copy (and tape it by your workstation!)` just for a more complete story of how it would be used!

Reply:
ooh, I like it! Will add 👍